datasetIngestor/addAttachment.go

```diff
@@ -11,27 +11,28 @@ import (
 	"strings"
 )
 
-func AddAttachment(client *http.Client, APIServer string, datasetId string, metaDataDataset map[string]interface{}, accessToken string, attachmentFile string, caption string) {
-	// turn image into base64 structure:
+func ReadAndEncodeImage(attachmentFile string) (string, error) {
 	imgFile, err := os.Open(attachmentFile)
 	if err != nil {
-		log.Fatalf("Can not open attachment file %v \n", attachmentFile)
+		return "", err
 	}
-
 	defer imgFile.Close()
-
+	
```

Comment:
I wonder why the linting doesn't complain here

Reply:
I thought this wasn't just an empty line, but an empty line with an extra `tab`

Reply:
it is with a tab. But all my code is like that. That's how vscode does it by default

Reply:
ok, that's weird, I thought lint would complain about that. Well not a problem anyway...